bip-p2qrh.mediawiki

```diff
@@ -65,25 +69,27 @@ It is proposed to use SegWit version 3. This results in addresses that start wit
 
 The proposal above also leaves a gap in case it makes sense to use version 2, or bc1z, for implementation of other address formats such as those that employ Cross Input Signature Aggregation (CISA).
 
-The proposed P2QRH address format is not intended to be compatible with TapScript. Instead, it is proposed to have a separate address format called P2QR to be defined in a follow-on BIP. This is meant to provide similar functionality to current P2TR addresses, but will likely require considerably more effort. P2QRH should be available to users who do not wish to have their public key exposed on-chain upon receiving payment.
+P2QRH is meant to be implemented on top of P2TR, combining the security of classical Schnorr signatures along with post-quantum cryptography. This is called "hybrid cryptography", so that there's no regression in security should a vulnerability be found in one of the signature algorithms used. One key distinction between P2QRH and P2TR however is that P2QRH will encode for a hash of the public key. This is a significant change in how Taproot works, but is necessary to avoid exposing public keys on-chain in advance of attackers.
```

Comment:
```suggestion
P2QRH is meant to be implemented on top of P2TR, combining the security of classical Schnorr signatures along with post-quantum cryptography. This is a form of "hybrid cryptography" such that no regression in security is presented should a vulnerability exist in one of the signature algorithms used. One key distinction between P2QRH and P2TR however is that P2QRH will encode a hash of the public key. This is a significant change in how Taproot works, but is necessary to avoid exposing public keys on-chain in advance of attackers.
```

Great sentence length and flow.